doc/tutorials/running_deb_as_serv_vm.rst

```diff
@@ -44,88 +44,93 @@ Validated Versions
 ******************
 
 - **Debian version:** 10.0 (buster)
-- **ACRN hypervisor tag:** acrn-2019w35.1-140000p
-- **Debian Service VM Kernel version:** 4.19.68-84.iot-lts2018-sos
+- **ACRN hypervisor tag:** acrn-2020w40.1-180000p
+- **Debian Service VM Kernel version:** 5.4.43-PKT-200203T060100Z
 
 Install ACRN on the Debian VM
 *****************************
 
-1. Clone the `Project ACRN <https://github.com/projectacrn/acrn-hypervisor>`_ code repository:
+#. Clone the `Project ACRN <https://github.com/projectacrn/acrn-hypervisor>`_ code repository:
 
    .. code-block:: bash
 
       $ cd ~
       $ git clone https://github.com/projectacrn/acrn-hypervisor
       $ cd acrn-hypervisor
-      $ git checkout acrn-2019w35.1-140000p
+      $ git checkout acrn-2020w40.1-180000p
 
 #. Build and install ACRN:
 
    .. code-block:: bash
 
-      $ make BOARD=nuc7i7dnb FIRMWARE=uefi
+      $ make all BOARD_FILE=misc/vm_configs/xmls/board-xmls/whl-ipc-i7.xml SCENARIO_FILE=misc/vm_configs/xmls/config-xmls/whl-ipc-i7/industry.xml RELEASE=0
```

Comment:
You are using a Whiskey Lake board here, but the prerequisite section still talks about a Kaby Lake NUC, can you update it?

Reply:
update to Kaby Lake NUC